frontend/src/app/application-forms/temporary-outfitters/temp-outfitter-left-nav.component.ts

```diff
@@ -21,19 +21,19 @@ export class TempOutfitterLeftNavComponent implements OnInit, OnChanges {
   constructor(private applicationFieldsService: ApplicationFieldsService, private util: UtilService) {}
 
   getControlStatus(control: FormControl) {
```

Comment:
This looks like another place where typedoc would be helpful.

Reply:
Done